lint-staged.config.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  '**/*.{ts,tsx}': (stagedFiles) => [`eslint .`, `prettier --write ${stagedFiles.join(' ')}`],
```

Comment:
Make sure eslint runs only on staged files as well.

Reply:
How can I do this? Could you help me?

Reply:
Thanks teacher